deCODE/Submit_Jobs

```diff
@@ -0,0 +1,31 @@
+gcloud auth application-default login
+
+# 
+conda activate CPG
+
+# test chrM
+input="gs://cpg-tob-wgs-main/mt/v7.mt"
+outputDir="deCODE"
+chr="M" 
+analysis-runner --dataset sgs-somatic-mtn --access-level main  --output-dir "$outputDir" --description "deCODE" ./submit/sub_dataproc.py \
+    --script "./deCODE/deCODE_by_CHR.py --dataset '$input' --chrom chr${chr} --output deCODE_chr${chr}.vcf.bgz" \
+    --jobname deCODE_chr$chr
+
+# Not by chr
+input="gs://cpg-tob-wgs-test/mt/v7.mt"
+outputDir="deCODE_test"
+analysis-runner --dataset sgs-somatic-mtn --access-level test  --output-dir "$outputDir" --description "deCODE test" ./sub/sub_dataproc.py \
+    --script "./scripts/deCODE_test.py --dataset '$input' --output deCODE_test.vcf.bgz" \
```

Comment:
This script file doesn't seem to exist - has it been renamed to ./deCODE/deCODE.py?

Reply:
Sorry for the confusion Katie! I didn't realize that all files will be reviewed. This chunk is outdated and I have deleted this file.